src/bounded_channel.rs

```diff
@@ -12,6 +13,20 @@ pub enum SendError {
     Disconnected,
 }
 
+impl fmt::Display for SendError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            SendError::Disconnected => write!(f, "channel disconnected"),
+        }
+    }
+}
+
+impl From<SendError> for String {
+    fn from(error: SendError) -> String {
+        error.to_string()
+    }
+}
```

Comment:
I believe if you implement Display then you get `.to_string()` by default, do you need the explicit From/Into?